indy_node/test/nym_txn/test_nym_additional.py

```diff
@@ -8,19 +8,32 @@
 from plenum.common.exceptions import RequestRejectedException
```

Comment:
This file was formatted with black but no other changes were made. This should be reverted to reduce noise.